package/Linux/gateway/debian/postinst

```diff
@@ -3,14 +3,18 @@
 # System-wide package configuration.
 DEFAULTS_FILE="/etc/default/devolutions-gateway"
 
+if [ ! -f $DEFAULTS_FILE ]; then
+	/bin/touch $DEFAULTS_FILE
+fi
```

Comment:
> And although not utilized by the service file, this PR also adds the functionality to create `/etc/defaults/devolutions-gateway` file.

This is likely something that was overlooked in the past.
I don’t think the program is ever reading the file `/etc/default/devolutions-gateway`.
The system-wide configuration is found at `/etc/devolutions-gateway/gateway.json`, and is created by the command `devolutions-gateway --config-init-only`.